R/deleteThis.R

```diff
@@ -0,0 +1,1789 @@
+# these functions should be deleted. I couldn't find any usage in the modules or jasp-desktop
+
+jaspResultsStrings <- function() {
```

Comment:
This one is an "active" hack to get stuff from jaspResults translated. In JASP, jaspResults is not a package so it cannot use its own translation files

Reply:
I've moved it back.